pkg/keystone/const.go

```diff
@@ -0,0 +1,56 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package keystone
+
+const (
+	// ServiceName -
+	ServiceName = "keystone"
+	// ServiceAccount -
+	ServiceAccount = "keystone-operator-keystone"
+	// DatabaseName -
+	DatabaseName = "keystone"
+	// KeystoneDatabasePassword - ref name to get the keystone db password from osp-secret
+	KeystoneDatabasePassword = "KeystoneDatabasePassword"
+	// AdminPassword - ref name to get the admin password from osp-secret
+	AdminPassword = "AdminPassword"
+
+	// InputHashName -Name of the hash of hashes of all resources used to indentify an input change
+	InputHashName = "input"
+
+	// KeystoneAdminPort -
+	KeystoneAdminPort int32 = 35357
+	// KeystonePublicPort -
+	KeystonePublicPort int32 = 5000
+	// KeystoneInternalPort -
+	KeystoneInternalPort int32 = 5000
+
+	// AppSelector -
+	AppSelector = "app"
+
+	// KollaConfig -
+	KollaConfig = "/var/lib/config-data/merged/keystone-api-config.json"
+
+	// InitContainerCommand -
+	InitContainerCommand = "/usr/local/bin/container-scripts/init.sh"
```

Comment:
Are constants like these really needed though? Unless these commands are used in more than 1 place I sort of thing having them in const.go instead of the resulting packages makes them harder to follow.

Reply:
the idea was, when apply this to the other operators, its probably good to just have to do customization in a single place and not in several files. But sure, we could migrate those which are only used once to the place where its used.

Reply:
Its just that having the constants here makes the other files/jobs/etc hard to follow. I'm all for constants where they are shared and you want one place to edit them. I'm generally a fan of having source code be more readable and in this case a constant at the top of the actual file these are used in would better suit